docs/Using Any API/single-word-response.md

```diff
@@ -0,0 +1,145 @@
+---
+layout: nodes.liquid
+section: ethereum
+date: Last Modified
+title: 'Single Word Response'
+permalink: 'docs/single-word-response/'
+whatsnext:
+  {
+    'Multi-Variable Responses': '/docs/multi-variable-responses/',
+    'Fetch data from an Array': '/docs/api-array-response/',
+    'Large Responses': '/docs/large-responses/',
+    'Make an Existing Job Request': '/docs/existing-job-request/',
+    'API Reference': '/docs/chainlink-framework/',
+    'Contract Addresses': '/docs/decentralized-oracles-ethereum-mainnet/',
+  }
+metadata:
+  title: 'Single Word Response'
+  description: 'Learn how to make a GET request to an API from a smart contract, using Chainlink.'
+  image:
+    0: '/files/930cbb7-link.png'
+---
+
+## Overview
+
+This guide explains how to make an HTTP GET request to an external API from a smart contract, using Chainlink's [Request & Receive Data](/docs/request-and-receive-data/) cycle and then receive one single response.
+
+**Table of Contents**
+
+- [Single Word Example](#single-word-example)
+- [Response Types](#response-types)
+- [Setting the LINK token address, Oracle, and JobId](#setting-the-link-token-address-oracle-and-jobid)
+- [Make an Existing Job Request](#make-an-existing-job-request)
+
+## Single Word Example
+
+This example shows how to:
+
+- Fetch a single word response in one single call.
+
+[Cryptocompare GET /data/pricemultifull API](https://min-api.cryptocompare.com/documentation?key=Price&cat=multipleSymbolsFullPriceEndpoint) returns all the current trading info (price, vol, open, high, low ..Etc) of any list of cryptocurrencies in any other currency that you need. To check the response, you can directly paste the following URL in your browser `https://min-api.cryptocompare.com/data/pricemultifull?fsyms=ETH&tsyms=USD` or run this command in your terminal:
+
+```curl
+curl -X 'GET' \
+  'https://min-api.cryptocompare.com/data/pricemultifull?fsyms=ETH&tsyms=USD' \
+  -H 'accept: application/json'
+```
+
+The response should be similar to the following:
```

Comment:
```suggestion
The response should be similar to the following example:
```
Usually it is best to end these sentences with a noun. "...following code:" "...following example:" "...following command:"